base/common/LineReader.cpp

```diff
@@ -16,6 +16,10 @@ extern "C"
 }
 #endif
 
+#if defined(__clang__) && __clang_major__ >= 13
```

Comment:
This fixes an error:
```
semin-serg@semin-serg-ub:~/ClickHouse/build-clang-13$ ninja
		[0/2] Re-checking globbed directories...
		[2/2166] Building CXX object base/common/CMakeFiles/common.dir/LineReader.cpp.o
		FAILED: base/common/CMakeFiles/common.dir/LineReader.cpp.o 
		/usr/bin/clang++-13  -DBOOST_ASIO_STANDALONE=1 -DPOCO_ENABLE_CPP11 -DPOCO_HAVE_FD_EPOLL -DPOCO_OS_FAMILY_UNIX -DSTD_EXCEPTION_HAS_STACK_TRACE=1 -DUSE_REPLXX=1 -DWITH_COVERAGE=0 -I../contrib/sentry-native/include -Iincludes/configs -I../base/common/.. -Ibase/common/.. -I../base/glibc-compatibility/memcpy -I../contrib/cityhash102/include -I../contrib/cctz/include -isystem ../contrib/libcxx/include -isystem ../contrib/libcxxabi/include -isystem ../contrib/libunwind/include -isystem ../contrib/boost -isystem ../contrib/poco/Net/include -isystem ../contrib/poco/Foundation/include -isystem ../contrib/poco/NetSSL_OpenSSL/include -isystem ../contrib/poco/Crypto/include -isystem ../contrib/boringssl/include -isystem ../contrib/poco/Util/include -isystem ../contrib/poco/JSON/include -isystem ../contrib/poco/XML/include -isystem ../contrib/replxx/include -isystem ../contrib/fmtlib-cmake/../fmtlib/include -isystem ../contrib/libc-headers/x86_64-linux-gnu -isystem ../contrib/libc-headers -fdiagnostics-color=always -fsized-deallocation  -gdwarf-aranges -pipe -mssse3 -msse4.1 -msse4.2 -mpclmul -mpopcnt -fasynchronous-unwind-tables -falign-functions=32   -Wall -Wno-unused-command-line-argument  -fdiagnostics-absolute-paths -fexperimental-new-pass-manager -Werror -Wextra -Wframe-larger-than=65536 -Wpedantic -Wno-vla-extension -Wno-zero-length-array -Wno-c11-extensions -Wcomma -Wconditional-uninitialized -Wcovered-switch-default -Wdeprecated -Wembedded-directive -Wempty-init-stmt -Wextra-semi-stmt -Wextra-semi -Wgnu-case-range -Winconsistent-missing-destructor-override -Wnewline-eof -Wold-style-cast -Wrange-loop-analysis -Wredundant-parens -Wreserved-id-macro -Wshadow-field -Wshadow-uncaptured-local -Wshadow -Wstring-plus-int -Wundef -Wunreachable-code-return -Wunreachable-code -Wunused-exception-parameter -Wunused-macros -Wunused-member-function -Wzero-as-null-pointer-constant -Weverything -Wno-c++98-compat-pedantic -Wno-c++98-compat -Wno-c99-extensions -Wno-conversion -Wno-ctad-maybe-unsupported -Wno-deprecated-dynamic-exception-spec -Wno-disabled-macro-expansion -Wno-documentation-unknown-command -Wno-double-promotion -Wno-exit-time-destructors -Wno-float-equal -Wno-global-constructors -Wno-missing-prototypes -Wno-missing-variable-declarations -Wno-nested-anon-types -Wno-packed -Wno-padded -Wno-shift-sign-overflow -Wno-sign-conversion -Wno-switch-enum -Wno-undefined-func-template -Wno-unused-template -Wno-vla -Wno-weak-template-vtables -Wno-weak-vtables -O2 -g -DNDEBUG -O3  -fno-pie   -D OS_LINUX -nostdinc++ -std=gnu++2a -MD -MT base/common/CMakeFiles/common.dir/LineReader.cpp.o -MF base/common/CMakeFiles/common.dir/LineReader.cpp.o.d -o base/common/CMakeFiles/common.dir/LineReader.cpp.o -c ../base/common/LineReader.cpp
		/home/semin-serg/ClickHouse/base/common/LineReader.cpp:34:5: error: identifier '__d0' is reserved because it starts with '__' [-Werror,-Wreserved-identifier]
		    FD_ZERO(&fds);
		    ^
		/home/semin-serg/ClickHouse/contrib/libc-headers/x86_64-linux-gnu/sys/select.h:88:26: note: expanded from macro 'FD_ZERO'
		#define FD_ZERO(fdsetp)         __FD_ZERO (fdsetp)
		                                ^
		/home/semin-serg/ClickHouse/contrib/libc-headers/x86_64-linux-gnu/bits/select.h:35:9: note: expanded from macro '__FD_ZERO'
		    int __d0, __d1;                                                           \
		        ^
		/home/semin-serg/ClickHouse/base/common/LineReader.cpp:34:5: error: identifier '__d1' is reserved because it starts with '__' [-Werror,-Wreserved-identifier]
		/home/semin-serg/ClickHouse/contrib/libc-headers/x86_64-linux-gnu/sys/select.h:88:26: note: expanded from macro 'FD_ZERO'
		#define FD_ZERO(fdsetp)         __FD_ZERO (fdsetp)
		                                ^
		/home/semin-serg/ClickHouse/contrib/libc-headers/x86_64-linux-gnu/bits/select.h:35:15: note: expanded from macro '__FD_ZERO'
		    int __d0, __d1;                                                           \
		              ^
		2 errors generated.
		[7/2166] Building CXX object utils/keeper-bench/CMakeFiles/keeper-bench.dir/main.cpp.o
		ninja: build stopped: subcommand failed.
```